### PR TITLE
Two cleanup patches to tests/node

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: pkg/controller/node/node_controller_test.go [hunk 1]
```diff
@@ -157,29 +155,17 @@ func checkAction(expected, actual core.Action, t *testing.T) {
 		e, _ := expected.(core.CreateAction)
 		expObject := filterLastTransitionTime(e.GetObject())
 		object := filterLastTransitionTime(a.GetObject())
-
-		if !equality.Semantic.DeepEqual(expObject, object) {
-			t.Errorf("Action %s %s has wrong object\nDiff:\n %s",
-				a.GetVerb(), a.GetResource().Resource, diff.ObjectGoPrintDiff(expObject, object))
-		}
+		assert.Equal(t, expObject, object)
```
Comment (kikisdeliveryservice) [inline pkg/controller/node/node_controller_test.go]: Migth be worth including the diff as an args to asset.Equal since that's a helpful error message.

Comment (cgwalters) [reply pkg/controller/node/node_controller_test.go]: Note this in the commit message:

> ... does diffs automatically, etc.

Try it!  The diffs output are pretty nice.

This one you quoted indeed tries to make a more useful error message but...it's IMO not worth the extra verbosity, and trying to do that consistently requires a *lot* more typing when adding new unit tests.

Comment (kikisdeliveryservice) [reply pkg/controller/node/node_controller_test.go]: > Note this in the commit message:
> 
> > ... does diffs automatically, etc.
> 
> Try it! The diffs output are pretty nice.
> 
I'll have to check this out!  Missed the commit message. :)

**Comment by cgwalters** [comment]
Any further comments on this?

**Pull request merged**